components/cam/src/control/physconst.F90

```diff
@@ -215,12 +218,17 @@ subroutine physconst_readnl(nlfile)
             if (newmwdry)   write(iulog,*)'***       MWDRY     ',shr_const_mwdair,mwdry,'***'
             if (newrearth)  write(iulog,*)'***       REARTH    ',shr_const_rearth,rearth,'***'
             if (newtmelt)   write(iulog,*)'***       TMELT     ',shr_const_tkfrz,tmelt,'***'
+	    if (newomega)   write(iulog,*)'***       OMEGA     ',shr_const_omega,omega,'***'
```

Comment:
Not important, but this line seems to have a different indentation from the others.